libs/features/mas/commerce/src/service.js

```diff
@@ -43,7 +43,7 @@ async function activateService(config, dataProviders) {
     const settings = Object.freeze(getSettings(config));
     // Fetch price literals
     try {
-        literals.price = await fetchPriceLiterals(settings);
+        literals.price = fetchPriceLiterals(settings);
```

Comment:
I would not call it fetch since it is not fetch anymore.

Reply:
Right. Renamed to `getPriceLiterals`.